tests/script/test_serialization.py

```diff
@@ -187,209 +187,194 @@ async def test_serialization_deserialization(websocket, context_id,
 @pytest.mark.asyncio
 @pytest.mark.parametrize(
     "js_string, expected_serialized",
-    [
```

Comment:
this was a `yapf` automatic reformat. Effectively only `iterator` tests are changed.